src/components/filter-controls/AnchorHeader.tsx

```
@@ -0,0 +1,95 @@
+    className="flex items-center gap-2 px-3 py-1.5 rounded-full bg-white/10 dark:bg-black/40 border border-white/20 shadow-sm transition-all hover:bg-white/20 cursor-pointer"
+    onClick={onClick}
+  >
+    <IconComponent icon={icon} size={16} color={color} />
+    <span className="text-xs font-bold text-white uppercase tracking-wider">{label}</span>
+    <button
+      onClick={(e) => {
+        e.stopPropagation();
+        onRemove();
+      }}
+      className="p-0.5 rounded-full hover:bg-white/20 text-white/60 hover:text-white"
+    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)